source/_integrations/matter.markdown

```diff
@@ -333,6 +333,46 @@ The CSA operates a Distributed Compliance Ledger (DCL) which provides metadata f
 Notification of an OTA update for a Matter device
 </p>
 
+## Automating on a button press
+
+To start an automation triggered by a button press (or a button press pattern), follow these steps.
+
+### Prerequisites
+
+- You have a Matter device that takes button presses as inputs, such as a Tuo Smart Button, VTM31SN dimmer by Inovelli, or the Matter Pushbutton Module by Innovation Matters.
+- The device has been added to Home Assistant
+
+### To automate on a button press
```

Comment:
Let move this to the event entity documentation, and reference it from here. This is not specific to Matter and we should not start documenting generic usage on each integration page.